docs/gitbook/examples/template.md

```diff
@@ -0,0 +1,373 @@
+# Using the Template Application
+
+This walkthrough starts where the [Getting Started](../getting-started/README.md) guide left off.
+
+The template application may seem like a lot but it comes with an architecture that allows each browser to interact with the application using its own private chain, and shows the user how to join two chains together through redirects.
```

Comment:
We need a link to the PPC documentation about here as it explains why the template does what it does.